covalent_slurm_plugin/slurm.py

```diff
@@ -73,37 +75,87 @@ def __init__(
         address: str,
         ssh_key_file: str,
         remote_workdir: str,
-        poll_freq: int,
-        cache_dir: str,
         options: Dict,
+        poll_freq: int = 30,
+        cache_dir: str = str(Path("~/.cache/covalent").expanduser().resolve()),
+        do_cleanup: bool = True,
         **kwargs,
     ):
         super().__init__(**kwargs)
 
         self.username = username
```

Comment:
should use "or get_config(...)" for all options here

Reply:
Using config for getting options now.